### DISCUSSION TITLE
Angular (no Ionic) with capacitor - external server

### DISCUSSION THREAD

**Discussion by dmitriy-viniukov** [ionic-team/capacitor#3780] [Q&A]
Hi everyone, I am stuck with capacitor configuration for my angular app. 

The problem is that when I want to proxy my external server (I set androidScheme and hostname in capacitor config), all xhr requests return 200 and **shouldInterceptRequest** header. But when I proxy the same server running locally (I set url in capacaitor config), it works fine.

So I have these lines in my capacitor.config.json 
` "server": {
    "hostname": "my-domain.com",
    "androidScheme": "https"
  }` 

but never have a proper response.

Any help would be appreciated

**Comment by jcesarmobile** [comment]
`hostname` and `androidScheme` don't proxy anything, those just configure how the url used to load your local assets look like, so don't make it match something that exists online if you want to access the remote url.

**Reply by dmitriy-viniukov** [reply]
So can I somehow configure the proxy then? I want all urls like /v1/* to be forwarded to my-domain.com

**Reply by jcesarmobile** [reply]
no

**Reply by zsjyjishukong** [reply]
So, if I want to load the data from my server,I have to deploy my front-end project to the server, and then config `capacitor.config.json` `"server: { url: "my front-end url" }`? This app is just a shortcut to a website？In my opinion, it should not be this, but I tried it for two days, and it didn't work on Android.
